sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -330,90 +331,99 @@ export class MessageSession extends LinkEntity {
     );
   }
 
+  private _connectionLock: string = getUniqueName("messageSessionOpen");
```

Comment:
Why is this called _connectionLock, but the other one called _openLock?

Reply:
Hahha, I suppose it means at that particular moment in time it seemed more connection-y than open-y. I'm torn but I'm going to just rename it to _openLock for consistency and since _openLock was there first (even though that name was also chosen by me).

Reply:
Removed.